inc/class-command.php

```diff
@@ -235,6 +242,16 @@ protected function shell( InputInterface $input, OutputInterface $output ) {
 		return $this->run_command( 'vagrant ssh' );
 	}
 
+	/**
+	 * Command to pass a command in to the virtual machine.
+	 */
+	protected function exec( InputInterface $input, OutputInterface $output ) {
+		$command = implode( ' ', $input->getArgument( 'options' ) );
+		$command = escapeshellcmd( $command );
```

Comment:
I think this might want to be [escapeshellarg](https://www.php.net/manual/en/function.escapeshellarg.php) instead? We can then do `$command = escapeshellarg( "cd /chassis && $command" )` and pass that all to `-c`.